app/src/main/java/com/sun/unsplash_01/data/source/remote/PhotoDetailAPI.kt

```diff
@@ -0,0 +1,16 @@
+package com.sun.unsplash_01.data.source.remote
+
+import com.sun.unsplash_01.data.model.PhotoDetail
+import com.sun.unsplash_01.utils.Constant
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+interface PhotoDetailAPI {
+
+    @GET("/photos/{id}")
+    suspend fun getPhotoDetail(
+        @Path("id") id: String?,
+        @Query("client_id") keyApi: String = Constant.API_KEY
```

Comment:
ApiKey sao không viết trong interceptor rồi add vào retrofit để mỗi api không phải khai báo thêm query này nữa 
```suggestion
override fun getOkHttpClientBuilder(): OkHttpClient.Builder {
        val okHttpBuilder = super.getOkHttpClientBuilder()
        okHttpBuilder.addInterceptor { chain ->
            val request = chain.request().newBuilder()
                val originalHttpUrl = chain.request().url
                val url = originalHttpUrl.newBuilder().addQueryParameter("api_key", "your api key value").build()
                request.url(url)
            val response = chain.proceed(request.build())
            return@addInterceptor response
        }
        return okHttpBuilder
    }
```